cmd/root/logger.go

```diff
@@ -97,12 +100,12 @@ func initLogFlags(cmd *cobra.Command) *logFlags {
 	flags.Var(&f.output, "log-format", "log output format (text or json)")
 
 	// mark fine-grained flags hidden from global --help
-	flags.MarkHidden("log-file")
-	flags.MarkHidden("log-level")
-	flags.MarkHidden("log-format")
+	_ = flags.MarkHidden("log-file")
```

Comment:
Can we set it up to ignore all of these calls using `exclude-functions`? https://golangci-lint.run/usage/linters/#errcheck

Reply:
We could but I'm not sure we should. If function where it called returned an error, I'd return it.

I'd prefer if we at least logged it so that if it does fail it's visible (Can happen if that flag was removed but this line was forgotten).

Reply:
But we don't do it in this PR and likely new added flags will follow the same pattern. I think we either need to add logging here already or add them to exclude-functions